src/app/views/tournament/tournament.component.ts

```diff
@@ -21,10 +24,38 @@ export class TournamentComponent implements OnInit{
     this.state$ = this.activatedRoute.paramMap.pipe(map(() => window.history.state));
     this.state$.subscribe(state => {
       this.tournament = state as Tournament;
+      this.tournament.winner = 'Rock'
     })
   }
 
   get isAuthenticated() {
     return this.authService.isAuthenticated;
   }
+
+  onTabChange(event: MatTabChangeEvent) {
+    if (event.index === 4) {
+      this.fireConfetti();
+    }
+  }
```

Comment:
This doesn't makes sense, you aren't waiting the person to show the winners tab, you are waiting that a player is marked as winner in the last round, then, you can change to the winner tab or use a dialog